src/script.js

```diff
@@ -393,7 +397,9 @@ const getEmails = () => {
 	const isInBundleFlag = isInBundle();
 	const processedEmails = [];
 	const allLabels = new Set();
+	const tabs = getTabs();
 
+  let currentTab = tabs.length > 0 ? document.querySelector('.aAy[aria-selected="true"]') : false;
```

Comment:
Spacing inconsistency to the left of this line.

This could also be reduced to:
```javascript
let currentTab = tabs.length && document.querySelector('.aAy[aria-selected="true"]');
```
But that's more of a personal style / readability thing.